src/azul/__init__.py

```diff
@@ -1676,45 +1675,38 @@ def docker_image_gists_path(self) -> Path:
 
     allowed_v4_ips_term = 'allowed_v4_ips'
 
-    waf_rate_rule_name = 'RateRule'
-
-    waf_rate_alarm_rule_name = 'RateAlarmRule'
-
-    waf_rate_rule_period = 300  # seconds; this value is fixed by AWS
-
-    waf_rate_rule_retry_after = 30  # seconds
-
-    waf_rate_rule_limit = 1000
-
-    @attrs.frozen(auto_attribs=True, kw_only=True)
-    class FileDownloadLimit:
-        rate_limit: int
-        evaluation_window: int
-        assumed_request_concurrency: float
-
-        @classmethod
-        def parse(cls, s: str) -> Self:
-            rate, s = s.split('/')
-            window, concurrency = s.split('@')
-            return cls(rate_limit=int(rate),
-                       evaluation_window=int(window),
-                       assumed_request_concurrency=float(concurrency))
-
-        @property
-        def retry_after(self) -> int:
-            return round(self.evaluation_window /
-                         self.rate_limit *
-                         self.assumed_request_concurrency)
-
-    @property
-    def waf_file_download_limit(self) -> FileDownloadLimit | None:
-        value = self.environ.get('azul_waf_download_rate_limit')
-        if value is None:
-            return None
-        else:
-            return self.FileDownloadLimit.parse(value)
+    @attr.s(frozen=True, kw_only=True, auto_attribs=True)
+    class RateRule:
+        name: str
+        limit: int  # number of requests
+        period: int  # evaluation window in seconds
+        retry_after: int  # in seconds
 
-    assert 100 <= waf_rate_rule_limit <= 2_000_000_000  # mandated by AWS
+        def __attrs_post_init__(self):
+            assert 10 <= self.limit <= 2_000_000_000  # mandated by AWS
```

Comment:
```suggestion
            assert 10 <= self.limit <= 2_000_000_000, self  # mandated by AWS
```

Reply:
According to our updated policy, this should be a requirement (`R(…)`). I can elaborate more in PL if desired.